h2o-core/src/main/java/hex/ModelMojoWriter.java

```diff
@@ -241,4 +244,14 @@ private void writeDomains() throws IOException {
       finishWritingTextFile();
     }
   }
+
+  /** Create file that contains model details in JSON format.
+   * This information is pulled from the models schema.
+   */
+  private void writeModelDetails() throws IOException{
+    ModelSchemaV3 modelSchema = (ModelSchemaV3) SchemaServer.schema(-1, model).fillFromImpl(model);
+    startWritingTextFile("properties/modelDetails.json");
+    writeln(modelSchema.toJsonString());
+    finishWritingTextFile();
```

Comment:
~Conceptually, the finishWritingTextFile should be in `finally` block - but depends on context (= if `writeln` can throw and exception.~ 

I was looking into implementation - writeln just appends string to StringBuffer.